src/OrchardCore.Modules/OrchardCore.Apis.GraphQL/GraphQLMiddleware.cs

```diff
@@ -122,6 +124,8 @@ private async Task ExecuteAsync(HttpContext context)
             catch (Exception e)
             {
                 await _serializer.WriteErrorAsync(context, "An error occurred while processing the GraphQL query", e);
+                _logger ??= context.RequestServices.GetRequiredService<ILogger<GraphQLMiddleware>>();
```

Comment:
Why you didn't use constructor injection to get the `ILogger<GraphQLMiddleware>`?

Reply:
Well. Since we used its default constructor to add instances in previous versions, it doesn't seem to be needed anymore, so I'll tweak the code